test/replication-luatest/gh_11053_gc_consumer_record_lost_after_promote_and_restart_test.lua

```diff
@@ -0,0 +1,80 @@
+local t = require('luatest')
+local replica_set = require('luatest.replica_set')
+local server = require('luatest.server')
+
+local g = t.group()
+
+g.before_each(function(cg)
+    cg.replica_set = replica_set:new{}
+    cg.box_cfg = {
+        replication = {
+            server.build_listen_uri('master', cg.replica_set.id),
+            server.build_listen_uri('replica', cg.replica_set.id),
+        },
+        replication_timeout = 0.1,
+        replication_synchro_timeout = 120,
+        replication_synchro_quorum = 4,
```

Comment:
I forgot again why do we have here quorum 4, and all still works fine 😭. If it is not needed, then can we drop it? If it is needed exactly 4, then can you write a comment why, please?